recipes/gdk-pixbuf/all/conanfile.py

```diff
@@ -59,16 +51,8 @@ def configure(self):
         if self.options.shared:
             self.options["glib"].shared = True
 
-    def requirements(self):
-        self.requires("glib/2.73.0")
-        if self.options.with_libpng:
-            self.requires("libpng/1.6.37")
-        if self.options.with_libtiff:
-            self.requires("libtiff/4.3.0")
-        if self.options.with_libjpeg == "libjpeg-turbo":
-            self.requires("libjpeg-turbo/2.1.2")
-        elif self.options.with_libjpeg == "libjpeg":
-            self.requires("libjpeg/9d")
+    def layout(self):
+        basic_layout(self, src_folder="source")
 
     def validate(self):
         if self.options.shared and not self.options["glib"].shared:
```

Comment:
If you bump to Conan 1.52.0, be sure to update these from `self.options` to `self.info.options` in the `validate()` method.